src/docs/development/data-and-backend/state-mgmt/declarative.md

```diff
@@ -15,13 +15,19 @@ If you're coming to Flutter from an imperative framework (such as Android SDK or
 iOS UIKit), you need to start thinking about app development from a new 
 perspective. 
 
+如果你是从命令式编程框架（例如 Android SDK 或者 iOS UIKit）转到 Flutter 开发的话，你需要从新的角度开始考虑 APP 开发了。
+
 Many assumptions that you might have don't apply to Flutter. For example, in 
 Flutter it's okay to rebuild parts of your UI from scratch instead of modifying 
 it. Flutter is fast enough to do that, even on every frame if needed.
 
+许多你可能有的假设并没有应用在 Flutter 中。例如，在 Flutter 应用中你可以重新构建某一部分的用户界面，而不是修改它。 如果你需要的话，Flutter 甚至在每一帧上都可以很快做到这一点。
```

Comment:
```suggestion
因此，很多在命令式框架下的假设可能并不适用于 Flutter。例如，在 Flutter 应用中你可以重新构建某一部分的用户界面，而不是修改它。 如果你需要的话，Flutter 甚至在每一帧上都可以很快做到这一点。
```
have don't apply to 通常翻译为在 xxx 里并不适用。
另外，Flutter it's okay to rebuild parts of your UI from scratch instead of modifying it 此句翻译可以更好。